test/methods/bools.jl

```diff
@@ -0,0 +1,156 @@
+using Test, GeometryOps
+import GeoInterface as GI
+import GeometryOps as GO
+
```

Comment:
This file doesn't exist anymore in more current versions of the code, right?

Reply:
Ah its a rebase mistake, this was a old PR